### PR TITLE
fixed problem with multiple entries

### PR DESCRIPTION
**Pull request by linuxlurak**
used array_key_exists to check if node/array exists or not

### CODE REVIEW

File: sources/source-LocalTel_CH.module [hunk 1]
```diff
@@ -35,16 +36,18 @@ class LocalTel_CH extends superfecta_base {
...
-			        }
-		        }
-	        }
-        }
-        return($caller_id);
-    }
-
-}
+			if(array_key_exists(0, $array['entry']))
+			{$caller_id = $array['entry'][0]['title'];}
```
Comment (tm1000) [inline sources/source-LocalTel_CH.module]: Not following FreePBX code formatting standards

File: sources/source-LocalTel_CH.module [hunk 2]
```diff
@@ -35,16 +36,18 @@ class LocalTel_CH extends superfecta_base {
...
+			else{
+                           if(array_key_exists('title', $array['entry']))
+			   {$caller_id = $array['entry']['title'];}
```
Comment (tm1000) [inline sources/source-LocalTel_CH.module]: Not following FreePBX code formatting standards

File: sources/source-LocalTel_CH.module [hunk 3]
```diff
@@ -35,16 +36,18 @@ class LocalTel_CH extends superfecta_base {
...
+			   else{
+			   	$caller_id = null;
+				$this->DebugPrint('Could not parse xml-response from tel.search.ch');
+				}
```
Comment (tm1000) [inline sources/source-LocalTel_CH.module]: Not following FreePBX code formatting standards

**Comment by linuxlurak** [comment]
I'am a hobbyist. Thanks for your changes. Is the code ok now? Cheers!

**Comment by tm1000** [comment]
Well nothing changed. So The code is still the same.

**Pull request closed without merging**